corehq/apps/accounting/models.py

```diff
@@ -1281,7 +1281,7 @@ def new_domain_subscription(cls, account, domain, plan_version,
         if date_end is not None:
             future_subscriptions = future_subscriptions.filter(date_start__lt=date_end)
         if future_subscriptions.count() > 0:
-            raise NewSubscriptionError(_(
+            raise NewSubscriptionError(unicode(_(
                 "There is already a subscription '%(sub)s' that has an end date "
```

Comment:
Ah, I see `_` is a reference to `ugettext_lazy`. In this case you could use `ugettext` (and drop the `unicode` call) to do that.